lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+
+def initialize
+	@menu = Hash.new
+	@menu = {:beef_burger => 4.5,
+					 :chicken_burger => 4.5,
+           :egg_fried_rice => 3,
+           :popcorn_duck => 6.00,
+           :katsu_curry => 8.00}
+
+end
+
+def show_menu
+ menu_display = @menu.dup
+ menu_display
+
+end
+
+def item_price (x)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)